web/tests/hashchange.test.js

```diff
@@ -22,6 +22,9 @@ const overlays = mock_esm("../src/overlays");
 const popovers = mock_esm("../src/popovers");
 const recent_view_ui = mock_esm("../src/recent_view_ui");
 const settings = mock_esm("../src/settings");
+mock_esm("../src/settings_data", {
+    user_can_create_public_streams: () => true,
+});
 const stream_settings_ui = mock_esm("../src/stream_settings_ui");
```

Comment:
I think overall, as a mocking strategy, it'd be better to be not mocking this module, and instead just have all of our node tests initialize the `current_user` module in  a normal way. But I suppose this can be OK for now.